### ISSUE TITLE
add button plugins fields 

### ISSUE THREAD

**Issue by guillermohaad** [pluginsGLPI/fields#52]
Hi Friends, 
how's it going over there?

I have a question about of plugins fields?

as you could save multiple tabs custom fields. This would add more not save?

example:
![campos personalizados](https://cloud.githubusercontent.com/assets/12921805/17857057/ea2d16f0-6845-11e6-993e-236a5630c64f.png)

have multiple records of the same tabs such as a button New Register

**Comment by trasher** [comment]
I'm sorry... I do not understand the question... Could you rephrase please?

**Comment by orthagh** [comment]
no answer, closing
